index.html

```diff
@@ -1,13 +1,45 @@
-Marilyn Cristina Malpu Jimenez
-"About"
-I am from Ecuador, but I live in United States for over two years.
-I am 23 years old, I graduated from the University as an Architect, however I want to learn coding since then.
-"Experience"
-Ibarra, creating the first design and construction as an Architect. 
-Manta, volunteering. 
-Oklahoma.
-Cartersville.
-"Connect"
-https://github.com/Maly22
-www.linkedin.com/in/marilynmalpu
-https://www.facebook.com/cristina.malpu
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Marilyn's Portfolio</title>
+  
+  <meta charset="UTF-8">
+  <meta name="description" content="Portfolio website showcasing Marilyn's work and experience">
+</head>
+
+<body>
+  <h1>Marilyn Cristina Malpu Jimenez</h1>
+  
+  <section id="About">
+    <h2>About</h2>
+    <p>I am from Ecuador, but I live in the United States for over two years. I am 23 years old. I graduated from the University as an Architect, however I want to learn coding since then.</p>
+  </section>
+  
+  <section id="Experience">
+    <h2>Experience</h2>
+    <ul>
+      <li>Project Coordinator in Serviconstrucciones, Ibarra, Ecuador
+        <br>Supported design and preparation of plans based on sustainability initiatives for unfamiliar projects.</br>
+    </li>
+      <li>Architect Drafter in Armazon Construction Company
+        <br>Read construction documents to determine the Quantity, Type, Material and Layout of a private police office.</br>
+    </li>
+      <li>Program Director in D&J Construction Company, Latacunga, Ecuador
+        <br>Researched on land use and regulations for building and subdividing properties in countries such as Ecuador, Peru and Bolivia.</br>
+    </li>
+      <li>Architect Assistant in Decentralized Autonomous Government of the City of Tulcan, Tulcan, Ecuador
+        <br>Supported collaboration with the development of the Partial Plan for the Development Pole in the eastern part of the city, turning it into an industrial zone.</br>
+      </li>
```

Comment:
This is not how the `br` tag works. The `br` is used to add a line break and it's a self-closing tag. That means that you don't use it it pairs, like this:
```html
<br>Line 1</br>
<br>Line 2</br>
```
You use it as a single, self-closing tag, like this:
```html
Line 1 <br />
Line 2 <br />
Line 3
```
I don't think you need those line breaks here, anyway. One suggestion I have for you to better structure your experience list is to use a heading element for the title and a paragraph for the description. Like this:

```suggestion
      <li>
        <h4>Project Coordinator in Serviconstrucciones, Ibarra, Ecuador</h4>
        <p>Supported design and preparation of plans based on sustainability initiatives for unfamiliar projects.</p>
      </li>
      <li>
        <h4>Architect Drafter in Armazon Construction Company</h4>
        <p>Read construction documents to determine the Quantity, Type, Material and Layout of a private police office.</p>
      </li>
      <li>
        <h4>Program Director in D&J Construction Company, Latacunga, Ecuador</h4>
        <p>Researched on land use and regulations for building and subdividing properties in countries such as Ecuador, Peru and Bolivia.</p>
      </li>
      <li>
        <h4>Architect Assistant in Decentralized Autonomous Government of the City of Tulcan, Tulcan, Ecuador</h4>
        <p>Supported collaboration with the development of the Partial Plan for the Development Pole in the eastern part of the city, turning it into an industrial zone.</p>
      </li>
```